action/attach_disk.go

```diff
@@ -7,19 +7,21 @@ import (
 	bslcvm "github.com/cloudfoundry/bosh-softlayer-cpi/softlayer/vm"
 )
 
-type AttachDisk struct {
+type AttachDiskAction struct {
 	vmFinder   bslcvm.Finder
 	diskFinder bslcdisk.Finder
 }
 
-func NewAttachDisk(vmFinder bslcvm.Finder, diskFinder bslcdisk.Finder) AttachDisk {
-	return AttachDisk{
-		vmFinder:   vmFinder,
-		diskFinder: diskFinder,
-	}
+func NewAttachDisk(
```

Comment:
Yes, all the other are named with "Action" suffix. What about submit another PR to cover your comment?  Since it is a little hard to combine them into it. Thanks!